src/gap-skipper.js

```diff
@@ -0,0 +1,85 @@
+import Ranges from './ranges';
+
+let seeking = false;
+let consecutiveUpdates = 0;
+let timer = null;
+let playerState;
+let lastRecordedTime;
+let adaptiveSeeking;
+let player;
+let tech;
+let gapSkipper;
+
+gapSkipper = function(options) {
+  player = this;
```

Comment:
we should probably make this a class/object rather than using module local variables